Src/Common/RequestResponseHeaders.cs

```diff
@@ -8,12 +8,12 @@ internal static class RequestResponseHeaders
         /// <summary>
         /// Source instrumentation header that is added by an application while making http requests and retrieved by the other application when processing incoming requests.
         /// </summary>
-        public const string SourceInstrumentationKeyHeader = "x-ms-request-source-ikey";
```

Comment:
Makes sense. I guess we can go with **x-ms-request-source** and **x-ms-response-target**. That will make it generic. So the header content can change to include whatever it needs to include about the request source and dependency target in future.

Also I suggest instead of using `appid-v1:` as the versioning string, we use `corr_id-v1:` or simply `id-v1:` That also removes any indication that this is appId. 

So today we will have `corr_id-v1: {app Id here}`, tomorrow we may have `corr_id: { base64 encoded json containing appId and roleInstance}`

Let me know if you agree.